src/pages/en/reference/api-reference.mdx

```diff
@@ -222,6 +222,22 @@ Getting a cookie via `Astro.cookies.get()` returns a `AstroCookie` type. It has
 | `number`       | `() => number` | Parses the cookie value as a Number. Returns NaN if not a valid number.   |
 | `boolean`       | `() => boolean` | Converts the cookie value to a boolean.   |
 
+### `Astro.redirect()`
+In [Server-side Rendering](/en/guides/server-side-rendering/), `Astro.redirect()` allows you to redirect to another page. 
```

Comment:
```suggestion
In [server-side rendering](/en/guides/server-side-rendering/) mode, `Astro.redirect()` allows you to redirect to another page. 
```
Just made this consistent with the other wording, by adding the "mode" here, too!